src/reducers/SignIn.js

```diff
@@ -0,0 +1,28 @@
+import {
+  SIGN_IN_FAILURE,
+  CLEAR_ERROR,
+  SIGN_IN_LOADING
+} from '../actions/action.types';
+import initialState from '../store/initialState';
+
+export default function reducer(state = initialState.auth.signin, action) {
+  switch (action.type) {
+    case SIGN_IN_LOADING:
+      return {
+        ...state,
+        loading: action.payload,
+      };
+    case SIGN_IN_FAILURE:
+      return {
+        ...state,
+        error: action.payload.error.message,
+      };
+    case CLEAR_ERROR:
+      return {
+        ...state,
```

Comment:
I don't seem to find where you are dispatching the action that gets to trigger `CLEAR_ERROR`

Reply:
It is being dispatched in the signin component at ```src/containers/Signin/Signin.js``` at ```handleChange``` and ```toggleModal``` functions.